dev/sparktestsupport/modules.py

```diff
@@ -170,6 +170,16 @@ def __hash__(self):
     ]
 )
 
+avro = Module(
```

Comment:
Why does this need a separate module unlike other datasources?

Reply:
This is much cleaner, like what we did for kafka, which is also a built-in data source. Ideally, we should separate parquet, orc and other built-in data sources from sql module. We can do the refactoring in the future, if needed